docs/Installation/Installation.md

```diff
@@ -226,16 +226,28 @@ To build SpECTRE with Singularity you must:
    will be dropped into a bash shell.
 5. `cd` into `WORKDIR/SPECTRE_HOME` and run `mkdir build && cd build` to set up
    a build directory.
-6. To build SpECTRE run
+6. To build SpECTRE, run
    ```
    cmake -D CMAKE_Fortran_COMPILER=gfortran-8 \
          -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-gcc
          WORKDIR/SPECTRE_HOME
    ```
-   followed by
-   `make -jN` where `N` is the number of cores to build on in parallel.
+   To build with clang, the CMake command is
+   ```
+    cmake -D CMAKE_CXX_COMPILER=clang++ \
+          -D CMAKE_C_COMPILER=clang \
+          -D CMAKE_Fortran_COMPILER=gfortran-8 \
+          -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-clang
+          WORKDIR/SPECTRE_HOME
+   ```
+   Compile the code with `make -jN` where `N` is the number of cores to build on
+   in parallel (e.g. `make -j4`).
+   * By default, `make -jN` without any specification will build all possible
```

Comment:
This is not true (or shouldn't be). We decided we don't want `make` (without a specific target) to build anything at all

Reply:
Sorry, this was clearly my mistake. I will delete that statement.
(From the output of `make list` I read that if no target is specified the default would be `make all`, from which my misunderstanding began)

`make -jN` seems to build a number of things. Below are what were built inside `/bin` after running `make` without target : 
```
ParallelInfo
RunTests
Test_AlgorithmBadBoxApply
Test_AlgorithmCore
Test_AlgorithmGlobalCache
Test_AlgorithmNestedApply1
Test_AlgorithmNestedApply2
Test_AlgorithmNodelock
Test_AlgorithmParallel
Test_AlgorithmReduction
Test_ConjugateGradientAlgorithm
Test_DistributedConjugateGradientAlgorithm
Test_DistributedGmresAlgorithm
Test_DistributedGmresPreconditionedAlgorithm
Test_DistributedRichardsonAlgorithm
Test_GlobalCache
Test_GmresAlgorithm
Test_GmresPreconditionedAlgorithm
Test_NewtonRaphsonAlgorithm
Test_RichardsonAlgorithm
Test_SchwarzAlgorithm
```

Reply:
the default target for make is whatever the first target is in the Makefile that does not begin with `.`

Reply:
The important thing, I think, is that we make no guarantee on what, if anything, will be built if you just run `make`. In the future I think it would be good to make the default target be something that tells you to build a specific target and how to get a list of available targets. If you want the unit tests, you have to build `unit-tests`, nothing else is guaranteed to produce a state that will pass the unit tests

Reply:
In this case default target is `all`, and yes, running `make all` builds the same list of executables as above, while building `unit-tests` seems to build only some of those. I thought `make all` would build _all_ possible targets, literally, at my first sight. @nilsdeppe so, at the current stage, does default target have no specific usage?

Reply:
We make no guarantee that `all` will build anything at all. In general, for a larger code base you almost never need or want to build everything. I think it's bad practice for the default/suggested behavior be "Build code for 45 minutes". The unit tests build quite a bit quicker and are usually all people need unless they actually want to run a specific executable. Does that make sense?

Reply:
Yes, understood. Thanks!